cookbooks/bcpc-hadoop/recipes/datanode.rb

```diff
@@ -18,6 +18,21 @@
   end
 end
 
+user_ulimit "hdfs" do
+  filehandle_limit 32769
```

Comment:
make the numbers configurable or we are always going to have these? If they're not going to be changed often, I wouldn't want an attribute for it.